Makefile

```diff
@@ -20,4 +20,4 @@ test-pyapp:
 
 .PHONY: test-pyramid-app
 test-pyramid-app:
-	@bin/make_test pyramid-app devdata=yes docker=yes frontend=yes postgres=yes
+	@bin/make_test pyramid-app devdata=yes docker=yes frontend=yes postgres=yes __postgres_port=5439
```

Comment:
Forcing the `__postgres_port` in the tests to be equal to the one used in `ci.yml` (otherwise it generates a random port when creating a new project)